packages/block/README.md

```diff
@@ -31,7 +31,7 @@ There are five static factories to instantiate a `Block`:
 - `Block.fromRLPSerializedBlock(serialized: Buffer, opts?: BlockOptions)`
 - `Block.fromValuesArray(values: BlockBuffer, opts?: BlockOptions)`
 - `Block.fromRPC(blockData: JsonRpcBlock, uncles?: any[], opts?: BlockOptions)`
-- `Block.fromEthersProvider(provider: ethers.providers.JsonRpcProvider | string, blockTag: string | bigint, opts: BlockOptions)`
+- `Block.fromJsonRpcProvider(provider: ethers.providers.JsonRpcProvider | string, blockTag: string | bigint, opts: BlockOptions)`
```

Comment:
By the current method signature of `Block`, this is an `any` and could thus be something else than `ethers.providers.JsonRpcProvider`

Reply:
Good catch.  I've added an `EthersProvider` interface in `util` that addresses this.